http-client/src/main/java/io/airlift/http/client/HttpClientConfig.java

```diff
@@ -444,4 +451,65 @@ public HttpClientConfig setTimeoutConcurrency(int timeoutConcurrency)
         this.timeoutConcurrency = timeoutConcurrency;
         return this;
     }
+
+    public String getLogPath()
+    {
+        return logPath;
+    }
+
+    @Config("http-client.log.path")
+    public HttpClientConfig setLogPath(String logPath)
```

Comment:
Add a config description explaining that this the files are named `<client-name>-http...`